test/unit/driver/pd_10_test.rb

```diff
@@ -1,3 +1,4 @@
+require 'rspec/collection_matchers'
```

Comment:
Just wondering - why did you add this line?

Reply:
Without this line this test file would not compile for me.

Reply:
Removed that line just to double check... Surprisingly, but now tests do not fail.